src/Jackett.Common/Definitions/newheaven.yml

```
@@ -200,9 +200,13 @@ search:
     grabs:
       selector: td:nth-child(7)
     seeders:
-      selector: td:nth-child(8)
+      selector: a[href*="#seeders"]
```

```suggestion
      selector: a[href$="#seeders"]
```

If it ends in this, we can use `$` instead.